src/main/java/com/example/home_work_8/geometry/Circle.java

```diff
@@ -0,0 +1,40 @@
+package com.example.home_work_8.geometry;
+
+public class Circle implements Geometry {
+
+    final static double pi = Math.PI;
+    private int circleS;
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+        this.circleS = figuraArea(radius, 0);
+    }
+
+    @Override
+    public int figuraArea(int radius, int height) {
+        int figurS = (int) pi * radius^2;
+        System.out.println("S circle: "+figurS);
```

Comment:
You must not use print statements in your code. The only exception is method `main` for the testing purposes